src/repositories/live.repository.ts

```diff
@@ -78,6 +80,20 @@ export class LiveRepository extends Repository {
     return body;
   }
 
+  public async getPostLiveViewerList(
```

Comment:
It is not repository method, it is `Feed` class. Please, look how [feeds](https://github.com/dilame/instagram-private-api/tree/master/src/feeds) works and refactor it.